applications/compressible_navier_stokes/turbulent_channel/application.h

```diff
@@ -246,7 +246,7 @@ class MyPostProcessor : public PostProcessor<dim, Number>
   {
     Base::do_postprocessing(solution, time, time_step_number);
 
-    statistics_turb_ch->evaluate(this->velocity, time, time_step_number);
+    statistics_turb_ch->evaluate(this->velocity.get(), time, time_step_number);
```

Comment:
I think this line is the only weakness of the current design. It always computes the velocity even though this might only be necessary every 10th/100th time step. Passing `SolutionField` to the statistics manager would be natural, but this is difficult since the same implementation is used for incompressible Navier-Stokes, where velocity is already a primary solution variable and introducing solution field there would mean additional implementation effort and somewhat needless complexity. So I can live with the current solution.